pkg/splunk/enterprise/configuration_test.go

```diff
@@ -611,6 +611,13 @@ func TestValidateAppFrameworkSpec(t *testing.T) {
 	var err error
 	ctx := context.TODO()
 
+	// remove the AppDownloadVolume if exist just to make sure
+	// previous test case have not created directory
+	err = os.RemoveAll(splcommon.AppDownloadVolume)
+	if err != nil {
+		t.Errorf("unable to delete directory %s", splcommon.AppDownloadVolume)
+	}
+
```

Comment:
As discussed, plz backup and restore, as that helps other test cases to expect a proper value.